src/NServiceBus.FileBasedRouting/FileBasedRoutingConfigExtensions.cs

```diff
@@ -11,8 +11,17 @@ public static class FileBasedRoutingConfigExtensions
         /// Enables routing configured with the routing configuration file.
         /// </summary>
         /// <param name="config">The configuration object.</param>
-        public static void UseFileBasedRouting(this RoutingSettings config)
+        /// <param name="updateInterval">The interval the route file should be checked for changes.</param>
+        public static void UseFileBasedRouting(this RoutingSettings config, TimeSpan? updateInterval = null)
```

Comment:
This is a breaking change. Should we add an overload instead of an optional param to avoid it?